R/gh_whoami.R

```diff
@@ -0,0 +1,74 @@
+#' Info on current GitHub user and token
+#'
+#' Reports wallet name, GitHub login, and GitHub URL for the current
+#' authenticated user, the first few and last characters of the token, and the
+#' associated scopes.
+#'
+#' Get a personal access token for the GitHub API from
+#' \url{https://github.com/settings/tokens} and select the scopes necessary for
+#' your planned tasks. The \code{repo} scope, for example, is one many are
+#' likely to need. The token itself is a string of 40 letters and digits. You
+#' can store it any way you like and provide explicitly via the \code{.token}
+#' argument to \code{\link{gh}()}.
```

Comment:
I haven't done this here. Maybe after the merge I could do it for all exported functions at once?